src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml

```diff
@@ -365,6 +366,17 @@
           p:component="null"
           p:parameterName="null"
           p:serviceImplForPurgeTableLookupCriteria-ref="concurEventNotificationLookupCriteriaPurgeService"/>
+          
+    <bean id="positionDataWorkdayLookupCriteriaPurgeService"
+          class="edu.cornell.kfs.sys.batch.service.impl.PositionDataWorkdayLookupCriteriaPurgeServiceImpl"/>
+          
+    <bean id="positionDataWorkdayTableDetailsForPurge" class="edu.cornell.kfs.sys.businessobject.TableDetailsForPurge"
+          p:businessObjectForRecordsTablePurge="edu.cornell.kfs.module.ld.businessobject.PositionDataWorkday"
+          p:useDefaultDaysBeforePurgeParameter="false"
+          p:nameSpaceCode="KFS-SYS"
+          p:component="PurgeTablesStep"
+          p:parameterName="PS_POSITION_DATA_WD_PARAMETER_NAME"
```

Comment:
Should this value be PS_POSITION_DATA_WD_NUMBER_OF_DAYS_OLD?

Reply:
Changing to match updated parameter name.